libs/use-publisher/src/utils.ts

```diff
@@ -0,0 +1,17 @@
+export const validateDuplicateSourceIds = (sourceId: string, allSourceIds: string[]) => {
```

Comment:
we should rename this function and have a comment to describe the function and returns